packages/runtime/datastore/src/localChannelContext.ts

```diff
@@ -217,19 +220,35 @@ export class RehydratedLocalChannelContext extends LocalChannelContextBase {
         return this.channel;
     }
 
-    private collectExtraBlobsAndSanitizeSnapshot(snapshotTree: ISnapshotTree, blobMap: Map<string, ArrayBufferLike>) {
+    private checkNeedToSanitizeAndCollectBlobs(
```

Comment:
The name of the function assumes action ("sanitize").
But in reality it's more generic as it tells if snapshot is of certain state.
It would be better to call it to reflect that, i.e. IsSnapshotInOldFormat or something like that